### DISCUSSION TITLE
Taking a composable screenshot with a time offset to skip initial animation

### DISCUSSION THREAD

**Discussion by alexandrucaraus** [cashapp/paparazzi#1558] [Q&A] [answered]
I see that there is an API for the classic views, which has an time offset to skip a few frames in order to let the initial animation to settle before taking a screenshot. But couldn't find a way for composables to do the same. Am I missing something?

**Comment by geoff-powell** [comment, marked as answer]
Currently the `paparazzi.snapshot { }` method does not support this. There are some issues logged to [fix](https://github.com/cashapp/paparazzi/issues/678) this in the future.
